src/quo2/core.cljs

```diff
@@ -255,6 +256,7 @@
 (def permission-tag quo2.components.tags.permission-tag/tag)
 (def status-tag quo2.components.tags.status-tags/status-tag)
 (def token-tag quo2.components.tags.token-tag/tag)
+(def network-tags quo2.components.tags.network-tags.view/network-tags)
```

Comment:
let's keep this alpha sorted - it's a huge file and this helps make it a bit easier to maintain 👍

Reply:
Done!